### ISSUE TITLE
[0.122.1] There was an error initializing DB: "syntax error at or near "s""

### ISSUE THREAD

**Issue by syphernl** [n8n-io/n8n#1847]
**Describe the bug**
After upgrading from `0.121.0` to `0.122.1` n8n fails to start:
```
2021-05-31 08:40:40 There was an error initializing DB: "syntax error at or near "s""
2021-05-31 08:40:40 Stopping n8n...
2021-05-31 08:40:35 UserSettings got generated and saved to: /home/node/.n8n/config
2021-05-31 08:40:35 Initializing n8n process
```

**To Reproduce**
1. Upgrade from `0.121.0` to `0.122.1`
2. See error

**Expected behavior**
No error

**Environment (please complete the following information):**
 - OS: `n8n container`
 - n8n Version: 0.122.1
 - Node.js Version `n8n container`
 - Database system PostgreSQL
 - Operation mode [e.g. own; operation modes are `own`, `main` and `queue`. Default is `own`]

**Additional context**

**Comment by janober** [comment]
Thanks a lot for reporting! Somebody else also reported the same issue here:
https://community.n8n.io/t/bug-n8n-instance-stopped-working-after-upgrading-from-0-121-0-to-0-122-1/6059/5

We are already on it but have sadly a hard time reproducing the issue.

**Comment by BHesseldieck** [comment]
Thanks for bringing this issue up! Can you maybe share which Postgres version you are using?

**Comment by janober** [comment]
Fix got released with n8n@0.122.2
